classes/TaxonomyUpload.php

```diff
@@ -193,6 +193,13 @@ public function loadFile($fieldMap){
 										$sciname .= ' '.$inputArr['unitname3'];
 									}
 								}
+								if(!empty($inputArr['cultivarepithet'])){
```

Comment:
I don't think that $inputArr['cultivarepithet'] and $inputArr['tradename'] have been standardized with single quotes and strtoupper() yet here, have they?

I.e., I think we need to process these strings similarly to the way that they are processed in classes/TaxonomyEditorManager.php.

Reply:
In other words, we should not assume that the end user is entering cultivarEpithet and tradeName with their correct formatting conventions before we concatenate them to sciname.

Reply:
Fixed in 360285873.